### ISSUE TITLE
Persistent excluded directories with 0.50.0

### ISSUE THREAD

**Issue by Gama11** [pinterest/ktlint#2100]
According to the CLI help, it sounds like the intended usage is to specify directories you want to ignore on the command line.

However, we found it more convenient to handle this via `.editorconfig` instead, then you have the exclusions in one place and can simply run `ktlint` without arguments from the project root directory:

```editorconfig
[{**/build/**/*.kts,**/build/generated-sources/**/*.kt}]
max_line_length = 9999
ktlint_standard = disabled
ktlint_experimental = disabled
```

Unfortunately this no longer works with 0.50.0, because "the `internal:ktlint-suppression` rule can not be disabled via the `.editorconfig`", so the output is spammed occurrences of `ktlint-disable` comments in `build/generated-sources/kotlin-dsl-accessors` files.

Am I missing something, is there a better way to handle this?

**Comment by paul-dingemans** [comment]
Tnx, for immediately trying the new release. This kind of usage was not anticipated. But clearly, it does not make sense to print this warning over and over again. Let me think about reducing it to one single (or at worst a couple of warnings due to parallel processing) warnings in the next release.

**Comment by Gama11** [comment]
Not sure it makes much of a difference whether it's 1 or 1000 warnings, it would result in a non-successful exit code either way.

**Comment by paul-dingemans** [comment]
> Not sure it makes much of a difference whether it's 1 or 1000 warnings, it would result in a non-successful exit code either way.
> 

Ah yes, fair point.

> Perhaps something like `.ktlintignore` is needed, similar to what other linters / formatters have?

I don't want to add another configuration file. Instead, it will become possible to disable ktlint entirely for a certain glob with one single property:

```
[{**/generated-sources/**/*.kts}]
ktlint = disabled
```
